samples/msal-browser-samples/VanillaJSTestApp2.0/app/default/authConfig.js

```diff
@@ -35,7 +35,8 @@ const msalConfig = {
 
 // Add here scopes for id token to be used at MS Identity Platform endpoints.
 const loginRequest = {
-    scopes: ["User.Read"]
+    scopes: ["User.Read"],
+    authenticationType: "pop"
```

Comment:
If you want to leave the parameter name above, this should be `authenticationScheme`

Reply:
done